go.mod

```diff
@@ -49,6 +49,7 @@ require (
 	github.com/lib/pq v1.3.0 // indirect
 	github.com/mailru/easyjson v0.7.1 // indirect
 	github.com/mattn/go-colorable v0.1.4 // indirect
+	github.com/mattn/goreman v0.3.5 // indirect
```

Comment:
can you run `go mod tidy`?

Reply:
Oops, sorry I didn't mean to commit `go.mod`. But since it's there, got it go-tidied up.